eng/scripts/Language-Settings.ps1

```diff
@@ -0,0 +1,100 @@
+$Language = "javascript"
```

Comment:
We should come up with better names so we know what $Language vs $Lang means.

Reply:
Ok, Am thinking of it like $Language is the full name of the language and $Lang is the abbreviation.
for e.g. $Language = "dotnet", "javascript" and $Lang = "net", "js"
Is there a better name I can use?

Reply:
We should figure out where it is used and name it after the usages.

Reply:
Its only the case for dotnet and js I noticed in the doc scripts e.g `update-docs-metadata.ps1` the language is specified in the different ways, I wanted to make both values available for use as the case may be.